data/db-schema/src/main/resources/net/corda/db/schema/vnode-crypto/migration/vnode-crypto-creation-v1.0.xml

```diff
@@ -2,26 +2,147 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
-    <changeSet author="R3.Corda" id="vnode-crypto-creation-v1.0">
+    <changeSet author="R3.Corda" id="crypto-creation-v1.0">
```

Comment:
Is there any reason `vnode` was removed from the ID? 
I'm guessing because it's a copy from the other on?

It may be clearer to have it on there to avoid confusion with the cluster one.

Otherwise, same comments as above.

Reply:
Yeah just a mistake syncing it up.

Having two separate copies is a pain, maybe I should have resisted this more